### DISCUSSION TITLE
Devices created in Domoticz by SBFspot have some inconsistencies

### DISCUSSION THREAD

**Discussion by Bertels999** [SBFspot/SBFspot#620] [General]
Helo, I just started using SBFspot on my raspberry PI using Domoticz. My PV convertor is a SMA Sunny Boy 2.5.

All works, but I have two minor suggestions for improvements.

Six devices are being created in Domoticz: Energy, AC voltage, Efficiency, Frequency, BT_Signal and Temperature.

1) The Energy device is configured by default with SubType kWh and Type Usage ("Verbruik" in Dutch). I had to adjust it manually to change the Type to supply ("Levering"). Not sure if this is an SBFspot thing or a Domoticz thing.

2) For the frequency a device is created with the SubType General and the Type percentage. However, the unit of frequency is Hz. 

Regards Bert

**Comment by sbf-** [comment]
> Not sure if this is an SBFspot thing or a Domoticz thing.

This is domoticz stuff... SBFspot has no types/subtypes defined
